tests/test_utils/test_serialize.py

```diff
@@ -140,28 +139,6 @@ def test_overlap_forbidden(self):
         ):
             save(Dummy, arrays={"val": [1]}, val=2)
 
-    @pytest.mark.requires_backend("tensorflow")
-    def test_tensorflow_support(self):
-        """Test that TensorFlow data is supported."""
-        x = math.astensor([1.1, 2.2])
-        loaded = load(save(DummyOneNP, name="myname", arrays={"array": x}))
-        assert tf.is_tensor(loaded.array)
-        assert np.array_equal(loaded.array, x)
-
-    @pytest.mark.requires_backend("tensorflow")
-    def test_backend_change_error(self, monkeypatch):
-        """Test that data must be deserialized with the same backend."""
-        x = math.astensor([1.1, 2.2])
-        path = save(DummyOneNP, name="myname", arrays={"array": x})
-        # can be thought of as restarting python and not changing to tensorflow
-        monkeypatch.setattr("mrmustard.math._backend._name", "numpy")
-        with pytest.raises(
-            TypeError,
-            match="Data serialized with tensorflow backend, cannot deserialize to the currently active numpy backend",
```

Comment:
Does this kind of error not occur between `numpy` and `jax`?

Reply:
If it can, this test should simply be updated, not removed, right?

Reply:
it should be identical if you just replace tensorflow with jax. The library [saves the backend](https://github.com/XanaduAI/MrMustard/blob/0a947b454ad095426e66566122ed2af2700c0ebb/mrmustard/utils/serialize.py#L71) when serializing an object, then [ensures you have the same backend active](https://github.com/XanaduAI/MrMustard/blob/0a947b454ad095426e66566122ed2af2700c0ebb/mrmustard/utils/serialize.py#L116) when deserializing that object.